### PR TITLE
i18n: replace sync for dt hunts

### PR DESCRIPTION
**Pull request by Souma-Sumire**
- I couldn't find the script, so this is done manually.
- Not sure if the German part of 'Kirlirger the Abhorrent' is correct.
- Not sure if the escape characters are handled correctly.

### CODE REVIEW

File: ui/raidboss/data/07-dt/hunts/heritage_found.ts [hunk 1]
```diff
@@ -162,7 +162,32 @@ const triggerSet: TriggerSet<Data> = {
...
     // ****** S-RANK: Atticus the Primogenitor ****** //
   ],
-  timelineReplace: [],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Heshuala': 'Heshuala',
+        'Urna Variabilis': 'Urna Variabilis',
+        'Atticus the Primogenitor': 'Atticus d(?:er|ie|as) Primogenitor',
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/hunts/heritage_found.ts:171]: It looks like we usually use `(?:der|die|das)` for the article here, if you wanted to keep with convention.

File: ui/raidboss/data/07-dt/hunts/living_memory.ts [hunk 2]
```diff
@@ -120,7 +120,32 @@ const triggerSet: TriggerSet<Data> = {
     },
     // ****** S-RANK: The Forecaster ****** //
   ],
-  timelineReplace: [],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Cat\'s Eye': 'Katzenauge',
+        'Sally the Sweeper': 'Sally d(?:er|ie|as) Fegerin',
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/hunts/living_memory.ts:128]: same comment as above.

File: ui/raidboss/data/07-dt/hunts/urqopacha.ts [hunk 3]
```diff
@@ -179,7 +179,32 @@ const triggerSet: TriggerSet<Data> = {
...
     // ****** S-RANK: Kirlirger the Abhorrent ****** //
   ],
-  timelineReplace: [],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Nechuciho': 'Nechuciho',
+        'Queen Hawk': 'Falkenkönigin',
+        'Kirlirger the Abhorrent': 'Kirlirger d(?:er|ie|as) Abscheuliche[rs]?',
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/hunts/urqopacha.ts:188]: same comment as above.

**Comment by Akurosia** [comment]
Generally looks good to me, only thing in my mind: usually we only translate stuff for which we have data and since non of them currently have s-rank data i am not 100% sure if we wanna keep the s-rank translations in it (i am fine with either tbh, just a generall question)

**Comment by Souma-Sumire** [comment]
I have no objections; you can freely edit this pull request to release the version.

**Pull request merged**